fiftyone/zoo/datasets/base.py

```diff
@@ -1157,6 +1158,98 @@ def _get_raw_dir(self, dataset_dir):
         return os.path.join(root_dir, "raw")
 
 
+class FIWDataset(FiftyOneDataset):
+    """Families in the Wild is a public benchmark for recognizing families via
```

Comment:
Small point: this description is shorter than the dataset's entry in the User Guide.

In the future, I'd like to remove the redundancy here by automatically generating the RST from these class docstrings (as is done for the model zoo). That will take a bit of work to achieve, but the point is that, if/when we do that, you might lose the longer description in the `.rst`.

tldr: consider updating this description to match the `.rst`.